docs/guide/installation-guide.txt

```diff
@@ -1,3 +1,128 @@
 ==================
 Installation Guide
 ==================
+
+The recommended way to get started using the Node.js driver is by using ``NPM`` (Node Package Manager) to install the dependency in your project.
+
+MongoDB Driver
+--------------
+
+After you've created your project with ``npm init`` , you can install the MongoDB driver and its dependencies with the command:
+
+.. code-block:: sh
+
+   npm install mongodb --save
+
+This will download the MongoDB driver and add a dependency entry in your ``package.json`` file.
+
+Troubleshooting
+---------------
+
+The MongoDB driver depends on several other packages, including:
+
+* `bson <https://www.npmjs.com/package/bson>`_
+* `require_optional <https://www.npmjs.com/package/require_optional>`_
+* `safe-buffer <https://www.npmjs.com/package/safe-buffer>`_
+* `saslprep <https://www.npmjs.com/package/saslprep>`_
+
+Additionally, there are multiple optional dependencies that can be installed alongside the driver:
+
+* `bson-ext <https://www.npmjs.com/package/bson-ext>`_
+* `kerberos <https://www.npmjs.com/package/kerberos>`_
+* `mongodb-client-encryption <https://www.npmjs.com/package/mongodb-client-encryption>`_
+* `snappy <https://www.npmjs.com/package/snappy>`_
+
+These optional modules are all native C++ extensions. They are optional extensions and are not required for the driver to function.
+
+bson-ext Module
+---------------
+
+The ``bson-ext`` module is an alternative **BSON** parser that is written in C++. If you wish to use the ``bson-ext`` module you will need to add the ``bson-ext`` module to your module's dependencies.
+
+.. code-block:: sh
+
+   npm install bson-ext --save
+
+kerberos Module
+---------------
+
+If you need to use ``kerberos`` module you will need to add the ``kerberos`` module to your module's dependencies.
```

Comment:
"You need more module, so you can module your module man!" :)

I think you mean to say "If you need support for connecting to an LDAP environment, you will need to add the kerberos module..."